apps/studio/pages/project/[ref]/observability/api-overview.tsx

```diff
@@ -52,6 +52,26 @@ export const ApiReport: NextPageWithLayout = () => {
     }
   }
 
+  const handleDatepickerChangeForRefresh = (vals: DatePickerValue) => {
+    handleDatePickerChangeFromHook(vals)
+    mergeParams({
+      iso_timestamp_start: vals.from || '',
+      iso_timestamp_end: vals.to || '',
+    })
```

Comment:
Can `to` & `from` be empty strings? if not I'd rather we use the `??` instead of `||`